assets/src/components/ladderPage.tsx

```diff
@@ -24,9 +38,53 @@ export const findSelectedVehicleOrGhost = (
   )
 }
 
-const LadderPage = (): ReactElement<HTMLDivElement> => {
-  const [state] = useContext(StateDispatchContext)
-  const { selectedRouteIds, selectedVehicleOrGhost } = state
+const LadderTab = ({
+  tab,
+  selectTab,
+}: {
+  tab: RouteTab
+  selectTab: () => void
+}): ReactElement<HTMLDivElement> => {
+  const title = tab.presetName || "Untitled"
+  return (
+    <div
+      className={
+        tab.isCurrentTab ? "m-ladder-page__tab-current" : "m-ladder-page__tab"
+      }
+      onClick={() => selectTab()}
+    >
+      {title}
+    </div>
+  )
+}
+
+const AddLadderButton = ({
```

Comment:
**nitpick**: from context it looks like this was supposed to be called `AddTabButton`

Reply:
Ah, yup, you're absolutely right.